### DISCUSSION TITLE
Scammer app

### DISCUSSION THREAD

**Discussion by Peete724** [community/community#102528] [Repositories]
### Select Topic Area

Product Feedback

### Body

Not sure if this was the right place for this topic, but here it is.

I found "Surver GetPayOff" app on Google Play some time ago and got scammed for about $180.
Developer uses github for the app (arshakn.github.io).
After some research I found out the app is flagged as scam by some users elsewhere. 
I've tried to reach out to the developer last 4-5 weeks without any replies and google play wont take it down because scamming is not against their policy for some reason.. idk. 

Hope you can do something about the person using Github scamming people.

**Comment by Akash1134** [comment]
Hi @Peete724 , 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- Reporting abuse or spam
- Reporting a user
- Reporting an issue or pull request
- Reporting a comment

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!

**Comment by MihaiCristianCondrea** [comment]
Hi, I wanted to add some information that might be relevant.

I recently encountered an app published on Google Play under the developer name [appmobdev](https://play.google.com/store/apps/developer?id=appmobdev) that redistributed my open-source Android application [Smart Cleaner for Android](https://play.google.com/store/apps/details?id=com.d4rk.cleaner) without complying with the GPLv3 license.

The app:

* reused my Play Store screenshots and icon (edited versions of them.. My man literally used Gemini),
* redistributed the compiled application,
* did not provide the corresponding source code required by the GPL license,
* and included my name in the app’s copyright section even though I did not authorize that distribution.

After documenting the issue, I submitted a copyright report to Google Play with screenshots and evidence. Shortly afterward the app was unpublished and the developer account appears to have been removed from the Play Store.

There are achieves on AppBrain about his account: https://www.appbrain.com/dev/appmobdev/
